BigEval.js

```diff
@@ -298,12 +295,34 @@ BigEval.prototype._tokenizeExpression = function (expression) {
 			i = parsed[1] - 1;
 			continue;
 		}
-		
+
+	    var isVarChars = this.varNameChars[c];
+
 		if (isVarChars) {
-			token += c;
+			// Starting a variable name - can start only with A-Z_
+			
+			var token = '';
+
+			while (i < len) {
+				c = expression[i];
+                isVarChars = this.varNameChars[c];
```

Comment:
I saw that our project was a mess (as seen in this line) when it came to indentation so I have fixed on following [happiness](https://github.com/JedWatson/happiness) guidelines in the project. 
Please rebase with master and then the new checks should apply on this PR too.